### ISSUE TITLE
Update node-sass versions.

### ISSUE THREAD

**Issue by Abhirup-99** [EOS-uiux-Solutions/eos-icons-landing#60]
The current node-sass version depends on 3.x node-gyp, which requires python2 for installation.

Python2 has reached EOL and installing the repo now requires an additional python2 installation.

The updated node-sass versions seem to fix the issue.

**Comment by chinmaym07** [comment]
Hey can I try this ?

**Comment by Abhirup-99** [comment]
Go ahead!

**Comment by chinmaym07** [comment]
Hey @Abhirup-99 . I tried upgrading this.
I have a query to which version should I update it ?  In the recent they say `node-sass` provides the same javascript api's as `sass`. [see here](https://sass-lang.com/blog/libsass-is-deprecated)
I tried using `sass`. it works fine. 
There are some warnings which are resolved .

![Screenshot from 2022-02-25 15-46-53](https://user-images.githubusercontent.com/44670961/155698640-9c4bc156-5310-4e6f-8084-2e36c954ee79.png)


I tried upgrading node-sass to 6.0.1 but faced this [issue](https://github.com/sass/node-sass/issues/3103) & couldn't resolve it ..
